### DISCUSSION TITLE
Buttons

### DISCUSSION THREAD

**Discussion by berot3** [mvdkwast/obsidian-copy-as-html#5] [Ideas]
Hi,

This might be a feature request, but can I have [buttons] on one note which copies the html of another note?

I imagine having one note with several buttons where I can copy the html of other notes. 

[buttons]: https://github.com/shabegom/buttons

**Comment by mvdkwast** [comment]
Hi, 

Maybe use something like [obsidian-macros](https://github.com/phibr0/obsidian-macros) or [Commander](https://github.com/phibr0/obsidian-commander) to create a macro that switches to the document and calls the *copy document to HTML* command ? 

I didn't test either of them, so not sure if this is possible. 

It's a bit out the scope of what I intended this plugin to do to provide an API for this kind of things.

**Reply by berot3** [reply]
Thanks a lot! I will dig around with what you suggested 👍🏻
